### PR TITLE
Remove need for calicoctl and CNI download URLs in master

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
@ozdanborne I think this is ready for a look.

**Comment by tomdee** [comment]
@caseydavenport Do you still want to get this PR in or can it be closed?

**Comment by caseydavenport** [comment]
@tomdee I'd still like to get this in.

I think I need to make some adjustments to it. I will try to get to it this week, otherwise we can close it.

**Comment by tomdee** [comment]
Deploy preview for *calico* ready!

Built with commit 199c3763e4762174dbe2c989499ca8f20fd8f538

https://deploy-preview-1920--calico.netlify.com

File: _includes/urls [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{%- if page.version == "master" -%}
```
Comment (ozdanborne) [inline _includes/urls]: shouldn't this also be `&& include.component == "calicoctl"`?

File: _includes/urls [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+https://projectcalico.org/builds/calicoctl
+{%- else -%}
```
Comment (ozdanborne) [inline _includes/urls]: would be cool if we could throw a runtime error here if an unexpected value is passed in for component. Maybe put a final `else` that tries to access a negative index of an array or something with a comment? Idk just an idea.

**Pull request merged**